### ISSUE TITLE
My First PyTorch Training - Rent Prediction

### ISSUE THREAD

**Issue by AgastyaPatel** [AgastyaPatel/FastAI-course#3]
Creating simple sequential Neural network; understand how it works and basic implementation of PyTorch

**Comment by AgastyaPatel** [comment]
The model seems to be quite good, Closing it until further updates.
![image](https://github.com/AgastyaPatel/FastAI-course/assets/69053403/48bee519-d103-4b94-b134-b291efb58b65)

**Comment by AgastyaPatel** [comment]
Current Accuracy 79%
